MIGRATION.md

```diff
@@ -1445,7 +1445,7 @@ We've also changed the code that resolves these globs, so that invalid globs wil
 Example of an **invalid** glob:
 
 ```
-stories: ['./**/*.stories.(ts|js)']
+stories: ['./**/*.stories.@(ts|js)']
```

Comment:
This is not an invalid glob

Reply:
ow lol, yeah I fixed the example of a invalid glob, lol